android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletCoinAdapter.java

```diff
@@ -241,11 +241,11 @@ public void setWalletCoinAdapterType(AdapterType type) {
     }
 
     public void setWalletListItemModelList(List<WalletListItemModel> walletListItemModelList) {
-        this.walletListItemModelList = walletListItemModelList;
+        this.walletListItemModelList = removeDuplicates(walletListItemModelList);
```

Comment:
I missed to notice this before but this should be done where the data is fetched or processed then passed the filtered data to UI. Ok for now as a workaround, probably address this in the next PR.

Reply:
It is causing an issue to show tokens from all networks so logged an issue https://github.com/brave/brave-browser/issues/28506